### PR TITLE
Feature/disk load save interceptor

### PR DESCRIPTION
**Pull request by daitangio**
A proposal fix for issue #6 
+ Add two load/save interceptor to support I/O PRG loading.
+ Minimal directory listing is supported
+ Support autostart: you can specify a filename, and startreal64 will load it from drive8, list it and run it

### CODE REVIEW

File: pyc64/realemulator.py [hunk 1]
```diff
@@ -52,10 +74,142 @@ def run_rom_code(self, reset):
...
+            for d in data:
+                listing.append(d)
+            listing.append(0)        
+        # For formatting see https://pyformat.info/#string_pad_align
+        add_line(0, "\u0012\"{:16.16}\" 00 2A".format("DRIVE"+str(deviceNumber)))
+        # scan directory and find out files.
+        # Then produce a "floppy disk drive"-like directory
+        total_blocks=0
+        for root, dirs, filenames in os.walk("./drive{}".format(deviceNumber)):
+            for fname in fnmatch.filter(filenames,"*.prg"):
```
Comment (irmen) [inline pyc64/realemulator.py:200]: .... why only *.prg?  I don't like the arbitrary restriction. Can you make it to just list all the files in the directory, regardless of the suffix?

**Review by irmen — CHANGES_REQUESTED** [review]

**Comment by irmen** [comment]
i like the fake input events to load a file from the command line argument :)

File: pyc64/realemulator.py (hunk 1 above)
Comment (daitangio) [reply pyc64/realemulator.py:200]: The code is not able to run bas file and I'd like to avoid showing them.
I will fix it in the branch.  :)

**Comment by daitangio** [comment]
> i like the fake input events to load a file from the command line argument :)
:) It is a dirty trick to speed up testing

File: pyc64/realemulator.py (hunk 1 above)
Comment (irmen) [reply pyc64/realemulator.py:200]: I see, fair enough.  When you save something from basic it will become a .prg anyway.  So it's fine as it is really

**Comment by irmen** [comment]
Thanks!

**Comment by daitangio** [comment]
Just fixed as suggested in the same branch. Done a bug fix on save, too.

**Pull request merged**